portal/cookbook-guide.md

```diff
@@ -258,6 +258,11 @@ As always, reach out to a Pythia team member for help with any of these steps!
 
 ### Initiate the Cookbook review process
 
```

Comment:
I'm wondering if this block of text needs to be integrated with the numbered list that follows this block. Immediately following this text block is the "submit a new cookbook" button, which I believe generates an issue on the authors repo containing the cookbook tasklist. It might be less confusing to mention the generation of the task list as part of the numbered list after submitting a new cookbook.